proto-compiler/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "tendermint-proto-compiler"
+version = "0.1.0"
+authors = ["Greg Szabo <greg@informal.systems>"]
+edition = "2018"
```

Comment:
It would be good to explicitly mention that this is not meant to be published (yet) via: https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field

Reply:
Opened an issue for that, good point.